### ISSUE TITLE
Save All in editor group not readable by screenreader

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#8657]
- VSCode Version: insiders 7/1
- OS Version: Windows

Screenreader can not read Save All button on an editor group in the explorer's Open Editors section.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/8658
